src/emc/usr_intf/gmoccapy/gmoccapy.py

```diff
@@ -551,9 +551,10 @@ def _get_pref_data(self):
         self.scale_feed_override = self.prefs.getpref("scale_feed_override", 1, float)
         self.scale_rapid_override = self.prefs.getpref("scale_rapid_override", 1, float)
 
-        # holds the max velocity value and is needed to be able to jog at
-        # at max velocity if <SHIFT> is hold during jogging
-        self.max_velocity = self.stat.max_velocity
+        # Holds the max velocity value and is needed to be able to jog at
+        # at max velocity if <SHIFT> is hold during jogging.
+        # The resulting jogging velocity will be automatically reduced to the maximum machine velocity. 
+        self.max_jog_velocity = 100
```

Comment:
Where does this get set to any other value? I think your reasonning is 100units/per second will be higher then system max and linuxcnc will limit it internally.
That assumption is not explicit nor future proof - 600 inches per minutes is totally doable.

You could get the actual value from getinfo.py 's function: get_max_jog_vel(self):

Reply:
> Where does this get set to any other value? I think your reasonning is 100units/per second will be higher then system max and linuxcnc will limit it internally. That assumption is not explicit nor future proof - 600 inches per minutes is totally doable.
> 


Yes it is not a very good solution, b I just did it like `continuous_jog()` is doing it to have the same behaviour like in Gscreen and other that uses this function. But it should be 100/60, it seems that I missed the "/60"

https://github.com/LinuxCNC/linuxcnc/blob/487e3de70b9f0487260de097f91615ed01e3cf69/src/emc/usr_intf/gscreen/emc_interface.py#L152-L165

https://github.com/LinuxCNC/linuxcnc/blob/487e3de70b9f0487260de097f91615ed01e3cf69/src/emc/usr_intf/gscreen/emc_interface.py#L23


> You could get the actual value from getinfo.py 's function: get_max_jog_vel(self):

Yes, that's much better, I didn't know. You should have mention that in https://github.com/LinuxCNC/linuxcnc/issues/1473 :wink: